feat(iam): add OIDC federations resources (#389)
selectel/terraform-provider-selectel: website/docs/r/iam_saml_federation_certificate_v1.html.markdown

Manages SAML Federation Certificates for Selectel products using public API v1.
Selectel products support Identity and Access Management (IAM).

> For more information about Federation Certificates, see the [official Selectel documentation](https://docs.selectel.ru/en/access-control/federations/certificates/).

Reviewer: ```suggestion
For more information about SAML Federation Certificates, see the [official Selectel documentation](https://docs.selectel.ru/en/access-control/federations/manage-saml/certificates/).
```

## Attributes Reference

> * `account_id` - Selectel account ID. The account ID is in the top right corner of the [Control panel](https://my.selectel.ru/). Learn more about [Registration](https://docs.selectel.ru/en/account/registration/).

Reviewer: ```suggestion
* `account_id` - Selectel account ID.
```

Author: Fixed!

where:

* `<account_id>` — Selectel account ID. The account ID is in the top right corner of the [Control panel](https://my.selectel.ru/). Learn more about [Registration](https://docs.selectel.ru/en/account/registration/).

* `<username>` — Name of the service user. To get the name, in the [Control panel](https://my.selectel.ru/iam/service-users), go to **Account** ⟶ the **Service users** tab ⟶ copy the name of the required user. Learn more about [Service Users](https://docs.selectel.ru/en/access-control/user-types/).

* `<password>` — Password of the service user.

> * `<federation_id>` — Unique identifier of the associated federation, for which the certificate is issued, for example, `abc1bb378ac84e1234b869b77aadd2ab`. To get the federation ID, use either [Control Panel](https://my.selectel.ru/iam/federations) or [Federations API](https://docs.selectel.ru/en/api/federations/).

Reviewer: ```suggestion
* `<federation_id>` — Unique identifier of the associated federation, for which the certificate is issued, for example, `abc1bb378ac84e1234b869b77aadd2ab`. Unique identifier of the federation, for example, `abc1bb378ac84e1234b869b77aadd2ab`. To get the federation ID, in the [Control Panel](https://my.selectel.ru/iam/federations), go to **Account** → **Federations** → copy the ID under the federation name.
```

Author: Fixed!
